lib/taskana-core-test/src/test/java/acceptance/workbasket/get/GetWorkbasketAccTest.java

```diff
@@ -23,18 +35,53 @@
 import pro.taskana.workbasket.api.models.Workbasket;
 import pro.taskana.workbasket.api.models.WorkbasketSummary;
 
-/** Acceptance test for all "get workbasket" scenarios. */
-@ExtendWith(JaasExtension.class)
-class GetWorkbasketAccTest extends AbstractAccTest {
+@TaskanaIntegrationTest
+public class GetWorkbasketAccTest {
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
 
-  private static final WorkbasketService WORKBASKET_SERVICE = taskanaEngine.getWorkbasketService();
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary =
+        WorkbasketBuilder.newWorkbasket()
+            .domain("DOMAIN_A")
+            .description("PPK User 2 KSC 1")
+            .name("PPK User 2 KSC 1")
+            .key("USER-1-2")
+            .type(WorkbasketType.PERSONAL)
+            .owner("user-1-2")
+            .orgLevel1("versicherung")
+            .orgLevel2("abteilung")
+            .orgLevel3("projekt")
+            .orgLevel4("team")
+            .customAttribute(CUSTOM_1, "custom1")
+            .customAttribute(CUSTOM_2, "custom2")
+            .customAttribute(CUSTOM_3, "custom3")
+            .customAttribute(CUSTOM_4, "custom4")
+            .buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-2")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.TRANSFER)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = defaultTestObjectReference().build();
+  }
 
   @WithAccessId(user = "user-1-2")
   @Test
   void should_ReturnWorkbasketWithId_When_IdIsValidAndUserHasPermissions() throws Exception {
-
-    Workbasket workbasket =
-        WORKBASKET_SERVICE.getWorkbasket("WBI:100000000000000000000000000000000007");
+    Workbasket workbasket = workbasketService.getWorkbasket(defaultWorkbasketSummary.getId());
 
     assertThat(workbasket.getDomain()).isEqualTo("DOMAIN_A");
```

Comment:
Please compare "workbasket" with "defaultWorkbasket" usind .isEqualTo(). Then, you won't need other assertions